packages/cli/lib/project-generator.js

```diff
@@ -131,13 +131,19 @@ module.exports = class ProjectGenerator extends BaseGenerator {
     });
   }
 
-  promptProjectName() {
+  promptProjectName(projectType) {
     if (this.shouldExit()) return false;
+    let namePrompt = 'Project name:';
+    let descriptionPrompt = 'Project description:';
+    if (projectType) {
+      namePrompt = `${projectType} name:`;
+      descriptionPrompt = `${projectType} description:`;
+    }
     const prompts = [
       {
         type: 'input',
         name: 'name',
-        message: g.f('Project name:'),
+        message: g.f(namePrompt),
```

Comment:
I don't think `g.f` works for parameters. Please globalize the string when assigning the parameters.

Reply:
+1, `g.f` does not support dynamically created inputs (AFAIK).

The best option is to keep two sets of strings, so that they can be fully translated to other languages.

```js
g.f('Project name');
g.f('Project description');
g.f('Application project name');
// ...
```

The second best option is to pass `projectType` as a parameter to `g.f`, but that means the project type will be always printed in English - not great for people who don't understand English!

```js
namePrompt = g.f('%s name:', projectType);
descriptionPrompt = g.f('%s description:', projectType);
```